reports/live-1-domains/main.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ministryofjustice/cloud-platform-environments/pkg/authenticate"
+	"github.com/ministryofjustice/cloud-platform-environments/pkg/ingress"
+	"github.com/ministryofjustice/cloud-platform-how-out-of-date-are-we/reports/pkg/hoodaw"
+	"k8s.io/api/networking/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+)
+
+// resourceMap is used to store both string:string and string:map[string]string key
+// value pairs. The HOODAW API requires the first entry of map to contain a string:string,
+// the rest of the map consists of a primary key (string) with a value containing a map (string:string)
+// of Kubernetes ingress resources that contain old live1-domain dns address names.
+type resourceMap map[string]interface{}
+
+var (
+	bucket         = flag.String("bucket", os.Getenv("KUBECONFIG_S3_BUCKET"), "AWS S3 bucket for kubeconfig")
+	ctx            = flag.String("context", "live.cloud-platform.service.justice.gov.uk", "Kubernetes context specified in kubeconfig")
+	hoodawApiKey   = flag.String("hoodawAPIKey", os.Getenv("HOODAW_API_KEY"), "API key to post data to the 'How out of date are we' API")
+	hoodawEndpoint = flag.String("hoodawEndpoint", "/live_1_domains", "Endpoint to send the data to")
+	hoodawHost     = flag.String("hoodawHost", os.Getenv("HOODAW_HOST"), "Hostname of the 'How out of date are we' API")
+	kubeconfig     = flag.String("kubeconfig", "kubeconfig", "Name of kubeconfig file in S3 bucket")
+	region         = flag.String("region", os.Getenv("AWS_REGION"), "AWS Region")
+
+	liveOneDomain = "live-1.cloud-platform.service.justice.gov.uk"
+	endPoint      = *hoodawHost + *hoodawEndpoint
+)
+
+func main() {
+	flag.Parse()
+
+	// Gain access to a Kubernetes cluster using a config file stored in an S3 bucket.
+	clientset, err := authenticate.CreateClientFromS3Bucket(*bucket, *kubeconfig, *region, *ctx)
+	if err != nil {
+		log.Fatalln(err.Error())
+	}
+
+	// Get all ingress resources
+	ingressList, err := ingress.GetAllIngressesFromCluster(clientset)
+	if err != nil {
+		log.Fatalln(err.Error())
```

Comment:
Same as above, just add some context to why it failed.

Reply:
Added a error message